papaparse.js

```diff
@@ -598,6 +598,7 @@ License: MIT
 		ChunkStreamer.call(this, config);
 
 		var xhr;
+		var timeout;
```

Comment:
XMLHttpRequest calls it "timeout". I prefer to use the same name.

Reply:
Personally, I'm not a fan of exposing bad naming from inner components to the outside world. Just because `XMLHttpRequest` has ambiguous names, does not mean that we have to 😊.

Just my 2 cent.

Reply:
Aside: seeing as this variable is not used anywhere but within the if-block of `if (this._config.downloadTimeout) {`, consider moving the variable into there.

Reply:
Fixed